testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/CliConfigTestCase.java

```diff
@@ -446,6 +446,96 @@ private void testTimeout(CliProcessWrapper cli, int config) throws Exception {
         assertEquals(getValue(cli.getOutput()), "" + config);
     }
 
+    @Test
+    public void testColorOutputViaConfigFile() throws Exception {
+        File f = createConfigFile(false, 0, true, false, true);
+        CliProcessWrapper cli = new CliProcessWrapper()
+                .setCliConfig(f.getAbsolutePath())
+                .addCliArgument("--controller="
+                                        + TestSuiteEnvironment.getServerAddress() + ":"
+                                        + TestSuiteEnvironment.getServerPort())
+                .addCliArgument("--connect");
+
+        try {
+            cli.executeInteractive();
+            cli.clearOutput();
+            cli.pushLineAndWaitForResults("read-attribute");
+            String out = cli.getOutput();
+            assertTrue(out.contains("Required argument --name is not specified."));
+            // Error message: red color, bold and high intensity with default background
+            assertTrue(out.contains("\u001B[1;91;109m") && out.contains("\u001B[0m\n"));
```

Comment:
I can see a \n, not sure this test will pass on Windows. Config.LINE_SEPARATOR is generally the way to go.